### DISCUSSION TITLE
Tracking for Aspire 9 support

### DISCUSSION THREAD

**Discussion by aaronpowell** [CommunityToolkit/Aspire#75] [General]
I've created https://github.com/CommunityToolkit/Aspire/milestone/1 which will track the various bits of work that is needed to integrate Aspire 9, and assigned some of the current issues to it.

As we hit things, either code that's brought over from the `dotnet/aspire` repo, or stuff that we have to comment out, let's create issues and add them to the milestone.

**Comment by aaronpowell** [comment]
The `main` branch is now using .NET Aspire 9 as the primary version to develop against.

**Comment by aaronpowell** [comment]
We're now building against the 9.0.0 GA packages for Aspire. There's a few outstanding PR's with some "last minute" integrations to tackle and then we'll cut the 9.0.0 release for the Toolkit.

@CommunityToolkit/aspire-contributors let's prioritise getting the currently open PR's done so we can aim for a release at the end of this week.

**Reply by Alirexaa** [reply]
#182 is a high priority
